apps/wallet/src/components/system-upgrade/RegistryUpdateMode.vue

```diff
@@ -134,7 +134,8 @@ const prepareUpdateWithWasmModule = async (): Promise<void> => {
 
     modelValue.value = {
       ...modelValue.value,
-      wasmModule: toArrayBuffer(artifact.artifact),
+      wasmModule: artifact.artifact.length ? toArrayBuffer(artifact.artifact) : undefined,
```

Comment:
what does this ternary operator do (in particular, why undefined)?

Reply:
In JS properties start as `undefined`, in this case if there is an artifact we add it, otherwise, we set as if the wasmModule was not set, hence the undefined.

Reply:
I see, thank you for the explanation!